src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -613,4 +615,27 @@ object DatabaseConversions {
       }
     }
   }
+
+  implicit val bakingRightsToRows = new Conversion[Id, (BlockHash, BakingRights), Tables.BakingRightsRow] {
+    override def convert(from: (BlockHash, BakingRights)): tezos.Tables.BakingRightsRow = {
+      val (blockHash, bakingRights) = from
+      bakingRights.into[Tables.BakingRightsRow]
+        .withFieldConst(_.blockHash, blockHash.value)
+        .withFieldRenamed(_.estimated_time, _.estimatedTime)
+        .transform
+    }
+  }
+
+  implicit val endorsingRightsToRows = new Conversion[Id, (BlockHash, EndorsingRights), Tables.EndorsingRightsRow] {
+    override def convert(from: (BlockHash, EndorsingRights)): tezos.Tables.EndorsingRightsRow = {
+      val (blockHash, endorsingRights) = from
+      endorsingRights.into[Tables.EndorsingRightsRow]
+        .withFieldConst(_.slots, endorsingRights.slots.mkString(","))
```

Comment:
Please re-use existing private methods to cleanly and consistently summarize collections to a string.
There's a method to make a csv, but consistently spacing values.

Reply:
removed with the other change